Makefile

```diff
@@ -252,16 +249,15 @@ CI_GOLANG_DOCKER_MAKE := $(DOCKER) run \
 	-e WASM_ENABLED=$(WASM_ENABLED) \
 	-e FUZZ_TIME=$(FUZZ_TIME) \
 	-e TELEMETRY_URL=$(TELEMETRY_URL) \
-	$(GOIMAGE) \
-	make
+	golang:$(GOVERSION)
 
 .PHONY: ci-go-%
 ci-go-%: generate
-	$(CI_GOLANG_DOCKER_MAKE) $*
+	$(CI_GOLANG_DOCKER_MAKE) /bin/bash -c "git config --system --add safe.directory /src && make $*"
 
 .PHONY: ci-release-test
 ci-release-test: generate
-	$(CI_GOLANG_DOCKER_MAKE) test perf wasm-sdk-e2e-test check
+	$(CI_GOLANG_DOCKER_MAKE) make test perf wasm-sdk-e2e-test check
```

Comment:
Would this not call `make make`? Since the `make $*` in CI_GOLANG_DOCKER_MAKE already has a `make` command..?

Reply:
The `make` was removed from `CI_GOLANG_DOCKER_MAKE` so `make ci-release-test` should work. `make  ci-go-ci-release-test` will not but it does not seem to work currently as well.

Reply:
🤦 Now I see it. Sorry for the noise. :+1: